docs/code-signing.rst

```diff
@@ -0,0 +1,10 @@
+Code signing policy
+===================
+
+The code used by the DebOps Project and commited in its repositories is
+expected to be signed by a valid GPG key of a DebOps Developer or DebOps
+Contributor. This should allow for secure code authentication and a valid audit
```

Comment:
@ypid Policy should contain some suggestions about key authentication. I imagine GPG keys signed by Developers should be fine, we could also rely on https://keybase.io/ and GitHub usage for Contributors.